sdk/servicebus/azure-servicebus/azure/servicebus/_common/constants.py

```diff
@@ -98,6 +98,9 @@
 _X_OPT_LOCK_TOKEN = b'x-opt-lock-token'
 _X_OPT_SCHEDULED_ENQUEUE_TIME = b'x-opt-scheduled-enqueue-time'
 
+MAX_USER_AGENT_LENGTH = 512
+USER_AGENT_PREFIX = "azsdk-python-servicebus"
```

Comment:
To help me understand; is this a formally defined value/in sync with how other services denote it?

Reply:
(Saw the thread in discussion, answered my own question :) )